### PR TITLE
Client: Server SSL Expiry added better error message. Fix: #3866

### PR DESCRIPTION
**Pull request by ThePhisch**
added a new exception to handle serverside ssl expiry with a better error message

### CODE REVIEW

File: lib/rucio/client/baseclient.py [hunk 1]
```diff
@@ -390,6 +390,7 @@ def _send_request(self, url, headers=None, type_='GET', data=None, params=None,
             self.logger.debug("Request data (length=%d): [%s]" % (len(data), text))
 
         result = None
+        SSLvalid: bool = True
```
Comment (mlassnig) [inline lib/rucio/client/baseclient.py:393]: I would flip it around, i.e. start from a level of non-trust

File: lib/rucio/common/exception.py [hunk 2]
```diff
@@ -1059,3 +1059,13 @@ def __init__(self, *args, **kwargs):
...
         self.error_code = 102
+
+
+class ServerSSLCertificateExpiredException(RucioException):
+    """
+    Server SSL Certificate has expired, auth is no longer possible
+    """
+    def __init__(self, *args, **kwargs):
+        super(ServerSSLCertificateExpiredException, self).__init__(*args, **kwargs)
+        self._message = 'Server SSL Certificate has expired, auth is no longer possible'
```
Comment (mlassnig) [inline lib/rucio/common/exception.py:1070]: if easy to do, add timestamps when the cert expired?

Comment (bari12) [reply lib/rucio/common/exception.py:1070]: @ThePhisch ?

Comment (ThePhisch) [reply lib/rucio/common/exception.py:1070]: it should do that

**Comment by ThePhisch** [comment]
this is an old PR, refer to #5919

**Pull request closed without merging**